src/Serializer/AbstractItemNormalizer.php

```diff
@@ -201,7 +201,10 @@ protected function setAttributeValue($object, $attribute, $value, $format = null
             return;
         }
 
-        $this->validateType($attribute, $type, $value, $format);
+        if (empty($context[AbstractObjectNormalizer::DISABLE_TYPE_ENFORCEMENT])) {
```

Comment:
Looks like the usage of the constant from the Serializer is causing failing tests, should the constant be included within this class (despite the duplication)?